src/main/java/org/jabref/logic/importer/fetcher/IEEE.java

```diff
@@ -115,7 +106,11 @@ private static BibEntry parseJsonRespone(JSONObject jsonEntry, Character keyword
         entry.setField(StandardField.ISBN, jsonEntry.optString("isbn"));
         entry.setField(StandardField.ISSN, jsonEntry.optString("issn"));
         entry.setField(StandardField.ISSUE, jsonEntry.optString("issue"));
-        entry.addFile(new LinkedFile("", Path.of(jsonEntry.optString("pdf_url")), "PDF"));
+        try {
+            entry.addFile(new LinkedFile(new URL(jsonEntry.optString("pdf_url")), "PDF"));
```

Comment:
This needs to be discused -> maybe in a separate PR?

The LinkedFile points to a file **locally stored**. Not to a URL. -- Maybe, just use the `URL` field of the BibEntry. Or use the URLDownloader to download and store the file. (Needs some investigation how JabRef downloads the file; since JabRef does some rename and folder-sort-magic)

Reply:
Linked files can also contain an online link.
However in this case I would set the URL field with the pdf url.

Reply:
I think this is correct: the url for the download of the fulltetx should be added as a linked file.

Reply:
Did not know this concept. Seemed to be introduced at 5e1adf8cfed29dc5ac01def881de4b4676e9a389. Thus, OK for me.